tests/framework/population/test_population_view.py

```diff
@@ -594,6 +593,27 @@ def test__format_update_and_check_preconditions_time_step_pass(
         assert result[col].equals(update[col])
 
 
+def test__format_update_and_check_preconditions_adding_simulants_replace_identical_data(
```

Comment:
I'm not familiar with test__...why the dunder?

Reply:
Because the method name itself has an underscore.